configuration/fisBomToFabric8MavenPlugin.fuse7.properties

```diff
@@ -8,4 +8,5 @@
 3.0.11.fuse-740029-redhat-00002=3.5.33.fuse-740029-redhat-00001
 3.0.11.fuse-750035-redhat-00001=3.5.33.fuse-750033-redhat-00001
 3.0.11.fuse-760025-redhat-00001=3.5.33.fuse-760025-redhat-00001
-3.0.11.fuse-770009-redhat-00002=3.5.42.fuse-770010-redhat-00001
\ No newline at end of file
+3.0.11.fuse-770009-redhat-00002=3.5.42.fuse-770010-redhat-00001
+3.0.12=4.3.0.fuse-780012-redhat-00001
```

Comment:
this is a big jump. Worth testing that the examples are still working with this new plugin. I do not think that it goes until deployment in automated tests

Reply:
unfortunately, it doesn't seem to work as-is with a fresh minishift install:

```
[ERROR] Failed to execute goal org.jboss.redhat-fuse:fabric8-maven-plugin:7.8.0.fuse-sb2-780038-redhat-00001:build (default) on project camel-ose-springboot-xml: Failed to execute the build: io.fabric8.maven.core.service.Fabric8ServiceException: Unable to build the image using the OpenShift build service
```

Reply:
my bad. This error was a misconfiguration. Must create the "project" for the user in OpenShift first. (the project name must correspond to the namespace parameter in the launch configuration.